app/assets/javascripts/intro.js

```diff
@@ -105,6 +107,45 @@ $(document).on('ready page:load turbolinks:load', function () {
     }
   });
 
+  // phone number inputs + validation
+  var phone_input = document.querySelector('#phone_input');
+  var emergency_phone_input = document.querySelector('#emergency_phone_input');
+
+  var iti_phone_input = window.intlTelInput(phone_input, {
+    preferredCountries: ['nl'],
+    separateDialCode: true,
+    hiddenInput: 'phone_number',
+    utilsScript: 'utils.js'
+  });
+
+  var iti_emergency_phone_input = window.intlTelInput(emergency_phone_input, {
+    preferredCountries: ['nl'],
+    separateDialCode: true,
+    hiddenInput: 'emergency_phone_number',
+    utilsScript: 'utils.js'
+  });
```

Comment:
This bit seems to be almost the same as [this bit](https://github.com/svsticky/constipated-koala/pull/493/files#diff-e6404b253eda092a7b6fec30c7aa84a8R7-R23), except for the id of the inputs. Can we deduplicate this?

Reply:
We can, no idea why I didn't do this